apps/dot_ext/admin.py

```diff
@@ -95,6 +107,24 @@ class MyAccessTokenAdmin(admin.ModelAdmin):
 admin.site.register(MyAccessToken, MyAccessTokenAdmin)
 
 
+class MyApprovalAdmin(admin.ModelAdmin):
+    list_display = ('uuid', 'user', 'application', 'created_at')
+    search_fields = ('uuid', 'user__username', 'application__name')
+    list_filter = ('uuid', 'user', 'application')
+    raw_id_fields = ('user', 'application')
+
+
+admin.site.register(MyApproval, MyApprovalAdmin)
+
+
+class MyAuthFlowUuidAdmin(admin.ModelAdmin):
+    list_display = ('code', 'auth_uuid')
+    search_fields = ('code', 'auth_uuid')
+
+
+admin.site.register(MyAuthFlowUuid, MyAuthFlowUuidAdmin)
```

Comment:
Does this have a chancel of overloading the admin UI similar to what tokens experience in the admin UI right now in production?

Reply:
@njdister The `MyApprovalAdmin` might have this issue. I noticed it was missing, so added an admin view similar to the `MyAccessTokenAdmin`. 
The `MyAuthFlowUuidAdmin` should be OK over time, since the entry is removed after it is used. They will just accumulate for incomplete auth flows.

Reply:
@njdister I removed the `MyApprovalAdmin` since it doesn't really go with this work. Will create a backlog Jira story related to adding this to the admin. Maybe in combination with reviewing/tuning the other model views with lots of records that are not performing well.